Day-II-Calculator/calculator/src/App.js

```diff
@@ -13,11 +13,11 @@ const App = () => {
       <NumberButton text="7" />
       <NumberButton text="8" />
       <NumberButton text="9" />
-      <NumberButton text="x" extraStyles="operator-styles" />
+      <NumberButton text="&times;" extraStyles="operator-styles" />
```

Comment:
Cool. I think I saw this tip on Wes Bos. Great job on getting the MVP done. For your description box, it'll be good to separate each task list. I see you have them up looking like on list, so just keep that in mind when you're almost done. And don't forget `Name - Project Title` on your PR title. 👍 💯 👍